lib/rucio/db/sqla/migrate_repo/versions/a118956323f8_added_vo_table_and_vo_col_to_rse.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013-2019 CERN for the benefit of the ATLAS collaboration.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Andrew Lister <andrew.lister@stfc.ac.uk>, 2019
+
+''' Added vo table and vo col to rse '''
+
+import datetime
+
+import sqlalchemy as sa
+from alembic import context
+from alembic.op import (add_column, create_table, create_unique_constraint,
+                        drop_column, drop_constraint, drop_table, bulk_insert)
+
+
+def String(*arg, **kw):
+    kw['convert_unicode'] = 'force'
+    return sa.String(*arg, **kw)
+
+
+# Alembic revision identifiers
+revision = 'a118956323f8'
+down_revision = '2cbee484dcf9'
+
+
+def upgrade():
+    '''
+    Upgrade the database to this revision
+    '''
+
+    if context.get_context().dialect.name in ['oracle', 'postgresql', 'mysql']:
+        schema = context.get_context().version_table_schema if context.get_context().version_table_schema else ''
+        # add a vo table
+        vos = create_table('vos',
+                           sa.Column('vo', String(3), primary_key=True),
+                           sa.Column('description', String(255)),
```

Comment:
`String(3)` looks too small for me. Maybe 50 or move the length to the constant file.

Reply:
Remark: If the VO name can change but needs to be referenced in other tables then the primary key should independent from the name like rse/id

Reply:
It's only 3 characters because these are used to prefix (Rucio internally, not externally) all accounts and scopes of that VO. If we would put a longer one that would potentially eat too much space of the scope and account.

Reply:
ok, it's a short vo name then. In a way, it looks more than an autogenerated id then. vo ->id , description -> vo and referenced as vo-id in the other tables if we follow the logic of some current objects imo. 

For illustration, what can be the column values for some of the current existing VOs, e.g., alice, atlas, ops, dteam, swegrid.se, ops.ndgf.org, nordugrid.org, skavengers.eu, eiscat3d ?